docs/openapi_v1.0.0-alpha.yaml

```
@@ -1415,6 +1462,4 @@ tags:
   - name: Suppliers
   - name: Products
   - name: Availability
-  - name: Reservations
-  - name: Cancellations
```

Going along with the above tagging, this should be retained in order to display the tag grouping correctly.

```suggestion
  - name: Cancellations
```